src/activity.ts

```diff
@@ -0,0 +1,40 @@
+import AinftBase from "./ainftBase";
+import { ActivityNftInfo, HttpMethod, NftActivityType, TaskTypeCategory } from "./types";
+
+export default class Activity extends AinftBase {
+  add(
+    appId: string,
+    userId: string,
+    data: any,
+    activityType: TaskTypeCategory | NftActivityType,
+    nftInfo?: ActivityNftInfo,
+  ) {
+    const body = {
+      appId,
+      userId,
+      data,
+      activityType,
+      nftInfo,
+    };
+    const trailingUrl = '';
+    return this.sendRequest(HttpMethod.POST, trailingUrl, body);
+  }
+
+  addNftRecord(
```

Comment:
what about adding some comments about how add() and addNftRecord() use cases are different?

Reply:
I added simple comments. Thanks!